src/tools/sss_userdel.c

```diff
@@ -109,8 +109,9 @@ static int kick_user(struct tools_ctx *tctx)
             }
         }
         if (child_pid == -1) {
+            ret = errno;
             DEBUG(SSSDBG_CRIT_FAILURE, "waitpid failed\n");
```

Comment:
Would you mind adding `errno` / `strerror(errno)` to this debug message please? (since we are touching this part of the code)

Reply:
Thank you for your review.
I added errno and strerror(errno) to DEBUG like "fork failed" in src/tools/sss_userdel.c.